tools/proto2python.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 PROTO_FOLDER='../cpprevolve/revolve/gazebo/msgs'
-GAZEBO_PROTO_FOLDER='/home/matteo/Tools/gazebo/include/gazebo-10/gazebo/msgs/proto/'
+GAZEBO_PROTO_FOLDER=:"${HOME}/installed/include/gazebo-10/gazebo/msgs/proto/"
```

Comment:
this line is probably wrong, needs to be adjusted for each computer. We could try using pkg-confing to find the correct folder location, or ask the user...

Reply:
gazebo as submodule?